change/@ni-spright-components-08e968ac-291f-4c66-949e-115ca77deef2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Unless beachball behaves differently the first time it runs on a package, I suspect this will prevent it from publishing. Don't we want to publish it? Or are you holding off for some future milestone?

I also think we want to establish the changelog files since those need to be checked into source.

Reply:
- Leaving as 0.0.1 is fine. Once the infra is set-up and ready for contributions we should bump it to 1.0.0.
- You don't need to check-in the empty changelog md and json files. The intention was that we should do a patch release so beachball starts generating them. I'm not sure if beachball will be confused by having the empty files so we can just remove them and let beachball handle them

Reply:
That was my misunderstanding / forgetting how beachball works. But Milan remembered better: I confirmed from some recent new library PRs in skyline that beachball can create it and we don't need it to be in this PR.